app/src/main/java/org/fossasia/openevent/app/core/auth/login/LoginFragment.java

```diff
@@ -64,6 +64,14 @@ public void onStart() {
         loginFragmentViewModel.getProgress().observe(this, this::showProgress);
         loginFragmentViewModel.getError().observe(this, this::showError);
         loginFragmentViewModel.getEmailList().observe(this, this::attachEmails);
+        loginFragmentViewModel.getIsTokenSent().observe(this, (value) -> {
+            if (value) {
```

Comment:
We usually try to keep the view logic free. What is the need of this if statement ? It seems like this can be done using a `SingleEventLiveData` that I added yesterday. Please check it out if it serves your purpose also.

Reply:
Yeah, Try to send the value to the view only when the return value from getIsTokenSent() is true. Then you'll not need to use the conditional statement here.

Reply:
@srv-twry Thanks for pointing out, the check can be removed as the fragment is recreated every time.